### DISCUSSION TITLE
Do we have India in locales?

### DISCUSSION THREAD

**Discussion by 23navi** [faker-js/faker#209] [Q&A] [answered]
I am unable to get fake Indian data for my project.
If it is then plz let me know the name and if it is not available, I will be happy to help add it.

**Comment by Shinigami92** [comment]
Try `id_ID`
If this not already worked, I will try to help when I'm not at mobile anymore.

**Reply by fzn0x** [reply]
Sorry @Shinigami92  id_ID is not indian but indonesian 😄

**Comment by 23navi** [comment, marked as answer]
Sorry I got it,  en_IND

**Reply by fzn0x** [reply]
Nice, thanks for coming back and telling us

**Reply by XiXiongMaoXiong** [reply]
Hindi is HI not en_IND

**Reply by 23navi** [reply]
> Hindi is HI not en_IND

Thanks but I was actually looking for Indian names in English.
I appreciate your reply.

**Reply by XiXiongMaoXiong** [reply]
> > Hindi is HI not en_IND
> 
> Thanks but I was actually looking for Indian names in English. I appreciate your reply.

You're welcome
